src/pages/Login/Login.js

```diff
@@ -1,7 +1,30 @@
 import React from 'react';
+import { useLocation } from 'react-router-dom';
+import User from './components/User';
 
 const Login = () => {
-  return <div>Login</div>;
+  const location = useLocation();
+  const currentURL = location.pathname;
+  const isSelectLogin = currentURL === '/login' && true;
```

Comment:
```suggestion
  const isSelectLogin = currentURL === '/login';
```
해당 조건 자체가 true or false의 값을 갖기 때문에
true를 다시 부여해주지 않아도 됩니다!